packages/server/graphql/mutations/helpers/makeSlackBlocks.ts

```diff
@@ -0,0 +1,35 @@
+export const makeSection = (text: string) => ({
+  type: 'section',
+  text: {
+    type: 'mrkdwn',
+    text
+  }
+})
+
+export const makeSections = (fields: string[]) => ({
+  type: 'section',
+  fields: fields.map((field) => ({
+    type: 'mrkdwn',
+    text: field
+  }))
+})
+
+type ButtonInput = {
+  text: string
+  url: string
+  type?: 'primary' | 'danger'
+}
+
+export const makeButtons = (buttons: ButtonInput[]) => ({
+  type: 'actions',
+  block_id: `actionblock${new Date().toISOString()}`,
```

Comment:
does this have to be a UID? if so, it won't be unique enough because this could execute twice in the same millisecond.
i'd just use `generateUID`